nni/contrib/compression/utils/evaluator.py

```diff
@@ -153,21 +149,19 @@ def _init_optimizer_helpers(self, pure_model: Module | pl.LightningModule):
         This is an internal API, ``pure_model`` means the model is the original model passed in by the user,
         it should not be the modified model (wrapped, hooked, or patched by NNI).
         That is, the optimizers & lr_schedulers obtained by ``Evaluator`` match the ``pure_model``.
-
         This function is used to record the status of the optimizers & lr_schedulers,
         and ensure NNI can reinitialize the optimizers & lr_schedulers with a similar but modified model.
-
         Notes
         -----
         This is a part of Evaluator initialization, please make sure this function has been called before using other evaluator functions.
         """
         raise NotImplementedError
 
-    def bind_model(self, model: Module | pl.LightningModule, param_names_map: Dict[str, str] | None = None):
+    def bind_model(self, model: Module | pl.LightningModule, param_names_map: Dict[str, str] | None = None, \
+                   module_name_param_dict: Union[Dict[str, List[Tensor]], None] = None):
```

Comment:
please add docstring, and I think put `module_name_param_dict` here will meet limitations, what if multi compressor need add param group but only one compressor can call `bind_model`?